index.html

```diff
@@ -179,6 +196,67 @@ <h3>
           }
         </pre>
       </section>
+      <section>
+        <h3>
+          Processing the `note_taking` member
+        </h3>
+        <p>
+          To <dfn>process the `note_taking` member</dfn>, given [=ordered map=]
+          |json:ordered map|, [=ordered map=] |manifest:ordered map|, [=URL=]
+          |manifest_URL:URL|, run the following during the
+          <a data-cite=
+          "appmanifest#dfn-processing-extension-point-of-web-manifest">extension
+          point</a> in [=processing a manifest=]:
+        </p>
+        <ol class="algorithm">
+          <li>If |json|["note_taking"] does not [=map/exist=], return.</li>
+          <li>If the type of |json|["note_taking"] is not [=ordered map=],
+            return.
+          </li>
+          <li>Set |manifest|["note_taking"] to a new [=ordered map=].</li>
+          <li>If |json|["note_taking"]["new_note_url"] does not [=map/exist=],
+            return.
+          </li>
+          <li>If the type of |json|["note_taking"]["new_note_url"] is not
+            [=string=], return.
+          </li>
+          <li>Let |new_note_url:URL| be the result of [=URL Parser|parsing=]
+            |json|["note_taking"]["new_note_url"] with |manifest URL| as the
+            base URL.
+          </li>
+          <li>If |new_note_url| is failure, return.</li>
+          <li>If |new_note_url| is not [=manifest/within scope=] of |manifest
+            URL|, return.
+          </li>
+          <li>Set |manifest|["note_taking"]["new_note_url"] to |new_note_url|.
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h3>
+          Launching the `new_note_url`
+        </h3>
+        <p>
+          To <dfn>launch the `new_note_url`</dfn>, given <a data-cite=
+          "appmanifest#dfn-processed-manifest">processed manifest</a>
+          |manifest:processed manifest|, run the following steps:
+        </p>
+        <ol>
+          <li>If |manifest|["note_taking"] does not [=map/exist=], return.</li>
```

Comment:
Combine these two (or three?) into one statement with "or"s.

Reply:
I actually think it's a bit cleaner to keep them all separate. Adding "or"s to makes it a more complex statement and is more likely to flow over multiple lines without nice code formatting.
Keeping as-is if that's ok.

Reply:
Ack.